### PR TITLE
Update use-device-guard-with-configuration-manager.md

### LINKED ISSUE #1489: Broken Link (use-device-guard-with-configuration-manager.md)
The File:
SCCMdocs\sccm\protect\deploy-use\use-device-guard-with-configuration-manager.md

Contains the following broken link:
https://docs.microsoft.com/windows/threat-protection/windows-defender-smartscreen/windows-defender-smartscreen-overview

### PR DESCRIPTION
**Pull request by AndrewJNet**
Fixed broken link filed in issue #1489

#MMS2019Docathon

### Summarize the change in the pull request title

Fixed broken link filed in issue #1489 

Fixes #1489

### CODE REVIEW

**Comment by PRMerger14** [comment]
@AndrewJNet : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
Thanks @AndrewJNet for the contribution! I'll review after the end of the contest.

#hold-off
(reviewers, please don't edit - I'll take care of it.)

File: sccm/protect/deploy-use/use-device-guard-with-configuration-manager.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ To verify the specific software being blocked or audited, see the following loca
 <!--Reworked article to put this inline while working on VSO 1355092
 ## Automatically let software run if it is trusted by Intelligent Security Graph
 
-You can let locked-down devices run software with a good reputation as determined by the Microsoft Intelligent Security Graph (ISG). The ISG includes [Windows Defender SmartScreen](https://docs.microsoft.com/windows/threat-protection/windows-defender-smartscreen/windows-defender-smartscreen-overview) and other Microsoft services. The devices must be running Windows Defender SmartScreen for this software to be trusted.
+You can let locked-down devices run software with a good reputation as determined by the Microsoft Intelligent Security Graph (ISG). The ISG includes [Windows Defender SmartScreen](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-smartscreen/windows-defender-smartscreen-overview) and other Microsoft services. The devices must be running Windows Defender SmartScreen for this software to be trusted.
```
Comment (aczechowski) [inline sccm/protect/deploy-use/use-device-guard-with-configuration-manager.md]: Remove locale (`en-us/`) from URL

**Comment by aczechowski** [comment]
Ping @AndrewJNet can you review and revise?

**Comment by PRMerger7** [comment]
@AndrewJNet : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
Thanks @AndrewJNet 
#sign-off 
*Once merged*, this should be live with the next daily publishing process. Right now that *should* be tomorrow (Wednesday, 22 May) around 11:00 Pacific time. No further action needed from you.

**Pull request merged**